lib/hammer_cli_katello/repository.rb

```diff
@@ -290,6 +290,17 @@ def request_params
       build_options :without => ["uuids"]
     end
 
+    class ExportCommand < HammerCLIKatello::Command
+
+      resource :repositories, :export
```

Comment:
the resource is already specified above, so you can just say ``action :export``.